package.json

```diff
@@ -40,11 +40,16 @@
     "ip": "1.1.5",
     "konfig": "0.2.1",
     "optional": "0.1.4",
+    "swagger-tools": "0.10.3",
     "uuid": "3.1.0",
     "vitalsigns": "0.4.3",
     "zoologist": "0.5.9"
   },
   "devDependencies": {
-    "nsp": "^3.1.0"
+    "chai": "^4.1.2",
```

Comment:
Would it be sensible to have set version numbers as per the dependencies in the devDependencies - just to make sure it's a level playing field?